targets/TARGET_NUVOTON/TARGET_M480/device/TOOLCHAIN_GCC_ARM/M487.ld

```diff
@@ -0,0 +1,295 @@
+/*
+ * Nuvoton M487 GCC linker script file
+ */
+
+#if !defined(MBED_APP_START)
+  #define MBED_APP_START 0x00000000
+#endif
+
+#if !defined(MBED_APP_SIZE)
+  #define MBED_APP_SIZE 0x00080000
+#endif 
+
+StackSize = 0x800;
+SPIM_CCM_START = 0x20020000;
+SPIM_CCM_END = 0x20028000;
+
+MEMORY
+{
+  VECTORS (rx)          : ORIGIN = MBED_APP_START, LENGTH = 0x00000400
+  FLASH (rx)            : ORIGIN = MBED_APP_START + 0x400, LENGTH = MBED_APP_SIZE - 0x00000400
+  RAM_INTERN (rwx)      : ORIGIN = 0x20000000, LENGTH = 0x00028000 - 0x00000000
+}
+
+/**
+ * Must match cmsis_nvic.h
+ */
+__vector_size = 4 * (16 + 96);
+
+ 
+/* Linker script to place sections and symbol values. Should be used together
+ * with other linker script that defines memory regions FLASH and RAM.
+ * It references following symbols, which must be defined in code:
+ * Reset_Handler : Entry of reset handler
+ *
+ * It defines following symbols, which code can use without definition:
+ * __exidx_start
+ * __exidx_end
+ * __etext
+ * __data_start__
+ * __preinit_array_start
+ * __preinit_array_end
+ * __init_array_start
+ * __init_array_end
+ * __fini_array_start
+ * __fini_array_end
+ * __data_end__
+ * __bss_start__
+ * __bss_end__
+ * __end__
+ * end
+ * __HeapLimit
+ * __StackLimit
+ * __StackTop
+ * __stack
+ */
+ENTRY(Reset_Handler)
+
+SECTIONS
+{
+    .isr_vector :
+    {
+        __vector_table = .;
+        KEEP(*(.vector_table))
+         . = ALIGN(4);
+    } > VECTORS
+
+    /* Note: Ensure that uVisor bss is at the beginning of SRAM to match vmpu_arch_init_hw of
+             uvisor/core/system/src/mpu/vmpu_armv7m.c */
+    /* Note: The uVisor expects this section at a fixed location, as specified
+             by the porting process configuration parameter: SRAM_OFFSET. */
+    __UVISOR_SRAM_OFFSET = 0x0;
+    __UVISOR_BSS_START = ORIGIN(RAM_INTERN) + __UVISOR_SRAM_OFFSET;
+    .uvisor.bss __UVISOR_BSS_START (NOLOAD):
+    {
+        . = ALIGN(32);
+        __uvisor_bss_start = .;
+
+        /* uVisor main BSS section */
+        . = ALIGN(32);
+        __uvisor_bss_main_start = .;
+        KEEP(*(.keep.uvisor.bss.main))
+        . = ALIGN(32);
+        __uvisor_bss_main_end = .;
+
+        /* Secure boxes BSS section */
+        . = ALIGN(32);
+        __uvisor_bss_boxes_start = .;
+        KEEP(*(.keep.uvisor.bss.boxes))
+        . = ALIGN(32);
+        __uvisor_bss_boxes_end = .;
+
+        /************************** ARMv7-M MPU only **************************/
+        __uvisor_bss_end_padding_max = (2 << (LOG2CEIL(__uvisor_bss_boxes_end - ORIGIN(RAM_INTERN)) - 1)) / 8;
+        . = __uvisor_bss_end_padding_max == 0 ? 0 :
+                MIN(__uvisor_bss_end_padding_max * (((__uvisor_bss_boxes_end - ORIGIN(RAM_INTERN)) / __uvisor_bss_end_padding_max) +
+                    MIN((__uvisor_bss_boxes_end - ORIGIN(RAM_INTERN)) % __uvisor_bss_end_padding_max, 1)) - __UVISOR_SRAM_OFFSET,
+                    ORIGIN(RAM_INTERN) + LENGTH(RAM_INTERN));
```

Comment:
@Patater I removed it based on uVisor porting guide. Please check https://github.com/ARMmbed/mbed-os/pull/4608/commits/c649e7d64a96b26ac83e8122f7038aba8dd38ab8.